doc/development/adding_operators.rst

```diff
@@ -315,7 +315,8 @@ implementations. The onus is on the contributors of new operators to add them to
 Adding your new operator to PennyLane
 #####################################
 
-Once the new operator is coded up, it is added to the appropriate folder in ``pennylane/ops/``. The
+If you want PennyLane to natively support your new operator, you have to make a Pull Request that adds it
+to the appropriate folder in ``pennylane/ops/``. The
```

Comment:
Minor one: I found it confusing because it sounds like any new custom operator _has_ to be added to PennyLane to be used, which is clearly not the case.